markov.js

```diff
@@ -1,4 +1,36 @@
-function buildChains(corpus, chainLength) {
+const fixCapitalisation = sentences => {
+    for (let i = 0; i < sentences.length; i++) {
+        sentences[i][0] = sentences[i][0].charAt(0).toUpperCase() + sentences[i][0].slice(1);
```

Comment:
I'm pretty sure we could generate (theoretically) a one character sentence (a), which could break stuff. It's pretty unlikely but still